tests/test_scraper_twitter.py

```diff
@@ -0,0 +1,35 @@
+"""
+This module contains the tests for the scraper.twitter module.
+"""
+
+import os
+import CryptoFraudDetection.utils.logger as logger
+from CryptoFraudDetection.scraper.twitter import TwitterScraper
+from CryptoFraudDetection.utils.enums import LoggerMode
+
+logger = logger.Logger(name=__name__, level=LoggerMode.DEBUG, log_dir="../logs")
+
+
+
+def test_initialization():
+    """
+    Test the initialization of the TwitterScraper class
+    """
+    username = os.getenv("TWITTER_USERNAME")
+    password = os.getenv("TWITTER_PASSWORD")
```

Comment:
you can set a custom value to test the initialization. You can also remove these inputs if username/password becomes optional (feedback in TwitterScraper.__init__). Then test if the default values (None) are set